script/Arcueid/Day 10/10_00_ARC10_7_2B.txt

```diff
@@ -530,6 +530,7 @@ The first thing that I feel is pure survival instinct.
 -- Page 802, Offset 26486.
 -- ここにいてはいけない。アレを見てはいけない。
 // This is very minor, but "mustn't" isn't really used in American English. This could be changed to "shouldn't" if it matters.
+// should we follow that comment above?
```

Comment:
I personally think must is more urgent but I'm also a filthy en_UK speaker

Reply:
:soubrit:
let's wait for the american's opinion then